hikari/api/rest.py

```diff
@@ -1130,7 +1412,29 @@ async def delete_webhook(
         *,
         token: undefined.UndefinedOr[str] = undefined.UNDEFINED,
     ) -> None:
-        ...
+        """Delete a webhook.
+
+        Parameters
+        ----------
+        webhook : hikari.snowflakes.SnowflakeishOr[hikari.webhooks.Webhook]
+            The webhook to delete. This may be a `hikari.webhooks.Webhook`
+            or the ID of an existing webhook.
+        token : hikari.undefined.UndefinedOr[builtins.str]
+            If specified, the webhoook token that will be used to delete
+            the webhook instead of the stored token.
```

Comment:
"stored token" isn't the most specific, could we phrase this to be more obvious that it's referring to the token this the client was initialised with?